DOCUMENTATION.md

```diff
@@ -490,7 +490,7 @@ A unique identifier you create. This reference identifies a single unique notifi
 
 #### pdfFile
 
-The precompiled letter must be a PDF file. For example:
+The precompiled letter must be a PDF file which meets [the specification](https://docs.notifications.service.gov.uk/documentation/images/notify-pdf-letter-spec-v2.3.pdf). For example:
```

Comment:
Suggest we change to the following to make it more descriptive:

The precompiled letter must be a PDF file which meets [the GOV.UK Notify PDF letter specification](https://docs.notifications.service.gov.uk/documentation/images/notify-pdf-letter-spec-v2.3.pdf)

Reply:
Update in 3b72b47e9da3ab2fa37007b0518a2f4e95bc7e73